spec/bundler/plugin_spec.rb

```diff
@@ -80,6 +80,38 @@
     end
   end
 
+  describe "uninstall command" do
+    before do
+      allow(installer).to receive(:uninstall).with(["new-plugin"]) do
+        { "new-plugin" => spec }
+      end
+    end
+    let(:opts) { {} }
+    it "remove plugin" do
+      allow(index).to receive(:remove_plugin).
+        with("new-plugin").once
+      allow(index).to receive(:installed?).
+        with("new-plugin").once
+      subject.uninstall ["new-plugin"], opts
+    end
+
+    context "multiple plugins" do
+      it do
```

Comment:
i don't encourage using `it` blocks without a description of what you're testing.